### PR TITLE
service control script added

### PR DESCRIPTION
**Pull request by guruswarupa**
# Pull Request

## Title

services control script
 - needs some working on add_service function. i'm not very clear on what all inputs are needed for a script to be made. it depends on what service we are creating but yeah i have made basic function. someone who's good at service management can improvise it.
 - developers can add new services script into services directory. which can then be used by other users using script.
- i've added i3lock-on-lid-close as a sample script which i made for my laptop to use when lid is closed.

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by guruswarupa** [comment]
> @adamperkowski man last pr broke by just 1 single line..... i had to re-add that -r which u told to remove.

and git rebase messed it up.

File: tabs/utils/service-control.sh [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    echo "Listing started services:"
+    $ESCALATION_TOOL systemctl list-units --type=service --state=running --no-pager | head -n -6 | awk 'NR>1 {print $1}' | more
+}
+
+# Function to add a new service
+add_service() {
+    while [ -z "$SERVICE_NAME" ]; do
+        echo "Enter the name of the new service (e.g., my_service):"
+        read -r SERVICE_NAME
+    done
```
Comment (adamperkowski) [inline tabs/utils/service-control.sh:58]: Should check if service already exists. Maybe something like this?

```suggestion
    while [ -z "$SERVICE_NAME" ]; do
        echo "Enter the name of the new service (e.g., my_service):"
        read -r SERVICE_NAME

        if $ESCALATION_TOOL systemctl list-units --type=service --all --no-legend | grep -q "$SERVICE_NAME.service"; then
            echo "Service already exists!"
            SERVICE_NAME=""
        fi
    done
```

**Review by adamperkowski — CHANGES_REQUESTED** [review]

**Comment by adamperkowski** [comment]
> and git rebase messed it up.

No problem.

File: tabs/utils/service-control.sh (hunk 1 above)
Comment (guruswarupa) [reply tabs/utils/service-control.sh:58]: is it working code?

Comment (adamperkowski) [reply tabs/utils/service-control.sh:58]: Technically, yes.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**